pkg/vpc/vpc_test.go

```diff
@@ -373,6 +376,52 @@ var _ = Describe("VPC - Import VPC", func() {
 			describeVPCError: nil,
 			error:            fmt.Errorf("invalid CIDR address: *"),
 		}),
+		Entry("VPC with secondary CIDR", importVPCCase{
```

Comment:
Can we make the `Entry` strings a little more descriptive? They don't tell me anything about the expected behaviour in this situation.

Reply:
Should they? I think of the entry strings as just describing the case under test and then the `Expect` calls/the rest of the test case as specifying the expected behavior. Like if it passes I think "VPCs with secondary CIDR works as expected" if it fails then I see where it fails to meet expectations.

Reply:
These strings in tests serve absolutely no purpose other than to describe what the code's behaviour, so it seems a shame not to make use of them. The perfect test file for me is one where I do not need to read the test code, or even the implementation code, to understand what is supposed to be happening. IMO when used properly they can make sharing context a dream and can really help people coming to a new area of code.

Reply:
I think they just describe which case is being tested. What if there's 20 `Expect` statements? The `DescribeTable` already has `"can import VPC"`. I have no problem changing them but I'm not sure how else to word this one...

Reply:
They don't need to be super verbose, like `"<in this case>, <does this thing>/<fails>/<does not fail>"`

Reply:
I get what you're saying but I think in these case we already have `DescribeTable("can import VPC", Entry("VPC with secondary CIDR", ...))`. I think the test case `struct` should tell me whether I'm expecting an error or it to work.

One problem is the `DescribeTable` description implies all the cases should work but some of them expect errors, I'm fixing the entire structure.

I've added a formatting function to improve the output.

Reply:
> I think the test case struct should tell me whether I'm expecting an error or it to work.

It _should_ but reading descriptions are faster and easier to line up outcome (pass/fail) with what was expected to happen.

> One problem is the DescribeTable description implies all the cases should work but some of them expect errors, I'm fixing the entire structure.

Yeh, then in this case it's fine if we expect everything in this bit to be "happy path". If there are some cases which error or in which the behaviour changes then it would be helpful in the description to point be clear on that.